app/src/main/java/com/mobiledevelopmentfoodapp/dto/Food.kt

```diff
@@ -4,7 +4,12 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Food(var name: String, var description: String, var price: Double, @SerializedName("id") var productId:Int = 0 ) {
+data class Food(
```

Comment:
Did the same for this data class, just re-organized and added trailing comma

Reply:
Duplicates a previous review.  When that happens, first in wins.